.github/scripts/test_extract_verdict.py

```
@@ -0,0 +1,261 @@
+class TestExtractVerdictScriptMain:
+    """Test the main() function when called via the script."""
+
+    def test_main_with_approve_verdict(self) -> None:
+        """Test main() returns 0 for APPROVE verdict."""
+        with tempfile.NamedTemporaryFile(mode="w", suffix=".md", delete=False) as f:
+            f.write("**APPROVE** — no issues")
+            f.flush()
+            temp_path = f.name
+
+        try:
+            result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)